LICENSE

```diff
@@ -1,6 +1,10 @@
-MIT License
+* The theme for this website is re-used under the MIT License. See file `LICENSE.txt` for more info.
```

Comment:
I think it's a bit confusing to have a separate `LICENSE.txt` file for the theme. Maybe we could include it into the end of this LICENSE file? It also looks like it contains Markdown, so it might make sense to combine everything into a `LICENSE.md` file.

Reply:
Good point!